addons/armory/stringtable.xml

```diff
@@ -46,8 +52,8 @@
             <German>Erstellen</German>
         </Key>
         <Key ID="STR_mts_Armory_checkbox_initializeEquipmentCategory_tooltip">
-            <Original>Initializes equipment category after creation and if possible adds action for everyone</Original>
-            <German>Initialisiert die Ausrüstungskategorie und fügt, falls möglich, eine Aktion für jeden hinzu</German>
+            <Original>If possible, adds interaction menu for the newly created equipment.\n(Depends if an object was given to the openArmoy function)</Original>
+            <German>Wenn möglich, fügt ein Interaktionsmenü für die neu erstellte Ausrüstungskategorie hinzu.\n(Abhängig davon, ob ein Objekt an die Funktion openArmoy übergeben wurde)</German>
```

Comment:
Not sure if `\n` is fine. Normally `<br />` is used.

```suggestion
            <Original>If possible, adds interaction menu for the newly created equipment.<br />(Depends if an object was given to the openArmoy function)</Original>
            <German>Wenn möglich, fügt ein Interaktionsmenü für die neu erstellte Ausrüstungskategorie hinzu.<br />(Abhängig davon, ob ein Objekt an die Funktion openArmoy übergeben wurde)</German>
```

Reply:
Not possible. The game uses the next correct stringtable entry if <br \> is used.